Sources/ComposableArchitecture/SwiftUI/Alert.swift

```diff
@@ -129,33 +129,38 @@ public struct AlertState<Action> {
 
   public struct Button {
     public var action: Action?
+    public var animation: Animation??
     public var type: `Type`
 
     public static func cancel(
       _ label: TextState,
-      send action: Action? = nil
+      send action: Action? = nil,
+      animation: Animation? = nil
```

Comment:
Ha in hindsight these arguments introduce some strange states, like `send: nil, animation: .default`. We have an idea on how to fix this but will do so in a follow-up PR.

Reply:
Yeah. This was something i've struggled with. Looking forward to the follow-up PR.